### PR TITLE
Feat/control channel

### LINKED ISSUE #39: [DJS-BOT] - Player control buttons
## Is your feature request related to a problem? Please describe.

> @BioCla One of the reasons I liked v5 is because it had buttons. It would be pointless to create a v5.1 without buttons
> 
> The player needs the buttons for they simplified the work in addition to the pause, play, stop, and volume commands
> 
> _Originally posted by @C-EO in https://github.com/SudhanPlayz/Discord-MusicBot/issues/1227#issuecomment-1599473535_
            

## Describe the ideal solution

> I'll take that as a suggestion. I was already planning on reimplementing them under a different dress:
> Instead of having it be on the player message itself it would be in a dedicated channel (kinda like hydra).
> 
> I'll try to reimplement it as you suggested as well
> 
> _Originally posted by @BioCla in https://github.com/SudhanPlayz/Discord-MusicBot/issues/1227#issuecomment-1599509202_

## Describe alternatives you've considered

None

## Additional context

Can't access them

### LINKED ISSUE #46: [DJS-BOT] - Switching DJ from user boolean property to Server Role property
https://github.com/wtfnotavailable/Discord-MusicBot/blob/199396490b81ce0e08be2ee2ba76dd20751c82b8/djs-bot/prisma/schema.prisma#L42C39-L42C39

As the title implies, further development and analysis has brought this issue to light.
I had mistakenly opted to save if a user was a DJ to the DB by flagging a boolean property, which isn't really ideal.
The proposed solution is to create a new `Guild` model within the current prisma schema which has a `DJRole` property of type `String?` which will be the ID of the role which be used to give special permissions to users using the bot.

### PR DESCRIPTION
**Pull request by Neko-Life**
## Draft
- [x] https://github.com/SudhanPlayz/Discord-MusicBot/issues/1173 
    - Create dedicated query channel system
- [x] https://github.com/wtfnotavailable/Discord-MusicBot/issues/39
    - Bring back add control buttons to now playing embed
- [x] https://github.com/wtfnotavailable/Discord-MusicBot/issues/46

## Please describe the changes this PR makes and why it should be merged:
Fix #39
Implements #46

## Status and versioning classification:
Merge into 5.2

### CODE REVIEW

File: djs-bot/lib/embeds.js
Comment (BioCla) [inline djs-bot/lib/embeds.js:1]: Move this to `djs-bot/util/`

**Comment by BioCla** [comment]
Awesome work! 👍

**Review by brianferri — APPROVED** [review]
ong, a legend!

**Comment by BioCla** [comment]
I was having so many problems with the help command ;-;

File: djs-bot/loaders/events/interactionCreate.js [hunk 1]
```diff
@@ -78,28 +78,36 @@ module.exports = async (client, interaction) => {
...
+
+			if (missingBotPerms.length)
+				missingPermsEmbed.addField("I'm missing some permissions:", `${missingBotPerms.join(", ")}`)
+
+			missingPermsEmbed.setFooter({ text: "If you think this is a mistake please contact the manager of this bot in this server." })
+
+			return interaction.reply({ embeds: [missingPermsEmbed], ephemeral: true })
+		}
+
+		// !TODO: what's this for?
```
Comment (BioCla) [inline djs-bot/loaders/events/interactionCreate.js:33]: Was left here from v4 iirc, supposed to consider sub commands but I never got to actually implementing them ehe

**Review by brianferri — DISMISSED** [review]
this is creating a conflict.
Since I pushed it directly to main branch, revert?

**Comment by BioCla** [comment]
For this conflict:
> djs-bot/tsconfig.json

Just take changes from main branch until actual ts is implemented

**Comment by Neko-Life** [comment]
is there anything more to do

**Comment by BioCla** [comment]
> is there anything more to do

Not for this PR i think.
Next steps are out of scope for this milestone

**Pull request merged**